vernac/g_vernac.mlg

```diff
@@ -475,8 +482,8 @@ GRAMMAR EXTEND Gram
   [ [ attr = LIST0 quoted_attributes ;
       bd = record_binder; rf_priority = OPT [ "|"; n = natural -> { n } ];
       rf_notation = decl_notations -> {
-      let (rf_canonical, rf_reversible), rf_locality = attr |> List.flatten
-        |> parse Notations.(canonical_field ++ reversible ++ option_locality) in
+      let (rf_canonical, rf_reversible), rf_locality =
+        parse_attr Notations.(canonical_field ++ reversible ++ option_locality) attr in
```

Comment:
BTW I can see it was already that way but we probably shouldn't be parsing attributes in the mlg

Reply:
Indeed, let's take the opportunity to fix that.